src/explore-education-statistics-common/src/hocs/withLazyLoad.tsx

```diff
@@ -6,7 +6,8 @@ interface LazyLoadProps<P> extends OmitStrict<BaseLazyLoadProps, 'children'> {
   placeholder?: ((props: P) => ReactNode) | ReactNode;
 }
 
-export default function withLazyLoad<P>(
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Doesn't seem like we need this. Removed it and there's no lint!

Unfortunately, there's a couple of other lints that do need fixing that are probably breaking the build